great-idea-website/index.html

```diff
@@ -19,69 +20,102 @@
-        Demand
+        <h1>Innovation<br>
+          On<br>
+          Demand</h1>
       </div>
       <div class="cta2">
         Get Started
       </div>
       <div class="cta3">
-      <img src="img/header-img.png" alt="Image of a code snippet.">
+        <img src="img/header-img.png" alt="Image of a code snippet.">
+      </div>
+    </section>
+
+    <hr>
+    <section class="mainContent">
+      <div class="mainContent1">
+        <div class="featuresBox">
+          <h2>Features</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+          </p>
+        </div>
+
+
+        <div class="aboutBox">
+          <h2>About</h2>
+          Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+          Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        </div>
       </div>
-      </section>
 
-<hr>
-      <section class="mainContent">
-        Features
-        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+      <div class="imageContainer">
+        <img class="middle-img" src="img/mid-page-accent.jpg" alt="Image of code snippets across the screen">
+      </div>
 
+      <div class="mainContent2">
 
-        About
-        Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+        <div class="servicesBox">
+          <h2>Services</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+          </p>
+        </div>
+
+        <div class="productBox">
+          <h2>Product</h2>
+          <p>Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+          </p>
+        </div>
+
+
+        <div class="visionBox">
+          <h2>Vision</h2>
+          <p>
+            Aliquam elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi.
+            Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.
+          </p>
+        </div>
+
+
+
+      </div>
+
+    </section>
+    <hr>
+    <address>
+      <h2>Contact</h2>
+      123 Way 456 Street<br>
+      Somewhere, USA<br>
+      <br>
+      1 (888) 888-8888<br><br>
+      sales@greatidea.io
```

Comment:
would recommend to use a <p> tag for each of this, <p>1(888)88888</p> and the same for email adress